src/yb/yql/pggate/pg_client.cc

```diff
@@ -770,6 +770,20 @@ class PgClient::Impl {
     return result;
   }
 
+  Result<client::YCQLStatStatementsInfo> YCQLStatStatements() {
+    tserver::PgYCQLStatStatementsRequestPB req;
+    tserver::PgYCQLStatStatementsResponsePB resp;
+
+    RETURN_NOT_OK(proxy_->YCQLStatStatements(req, &resp, PrepareController()));
```

Comment:
The number of entries is controlled by a gflag. At some point perhaps we should make it a request parameter.